static/src/javascripts/projects/commercial/modules/close-disabled-slots.js

```diff
@@ -21,25 +21,43 @@ const closeDisabledSlots = once((): Promise<void> => {
 });
 
 const mpuCandidateClass: string = 'fc-slice__item--mpu-candidate';
-const mpuHiderClass: string = 'fc-slice__item--no-mpu';
 const mpuCandidateSelector: string = `.${mpuCandidateClass}`;
 
-const shouldDisableAdSlotWhenAdFree = adSlot =>
+const shouldRemoveAdSlotWhenAdFree = () => commercialFeatures.adFree;
```

Comment:
I expect this may change, depending on GLabs / merchandising requirements. It's ok (I think) for this PR though.